src/components/ChatCustomAvatar.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {Image, View} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../styles/styles';
+import ActiveRoomAvatar from '../../assets/images/avatars/room.svg';
+
+const propTypes = {
+    /** Url source for the avatar */
+    source: PropTypes.string,
+
+    /** Extra styles to pass to Image */
+    imageStyles: PropTypes.arrayOf(PropTypes.object),
+
+    /** Extra styles to pass to View wrapper */
+    containerStyles: PropTypes.oneOfType([PropTypes.object, PropTypes.arrayOf(PropTypes.object)]),
+
+    /** Set the size of ChatCustomAvatar */
+    size: PropTypes.oneOf(['default', 'small']),
+
+    /** Whether this avatar is for an custom room */
+    isCustomChatRoom: PropTypes.bool,
+
+    /** Whether this avatar is for an custom room */
+    CustomChatRoomIcon: PropTypes.func,
+
+};
+
+const defaultProps = {
+    source: '',
+    imageStyles: [],
+    containerStyles: [],
+    size: 'default',
+    isCustomChatRoom: false,
+    CustomChatRoomIcon: ActiveRoomAvatar,
+};
+
+const ChatCustomAvatar = (props) => {
```

Comment:
There is an `Avatar` component that looks like this. Please use that and customize it for your needs. This component is breaking the app on the web while accessing the Rooms report.